dongkyu/initial-settings-express/myapp/db/migrations/20221221023540_create_userInfo_table.sql

```diff
@@ -0,0 +1,12 @@
+-- migrate:up
+CREATE TABLE userInfo (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  name VARCHAR(100) NOT NULL,
+  age INT NULL,
+  email VARCHAR(100) NOT NULL,
+  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP
+);
```

Comment:
- 테이블명은 복수형으로 작성하는 것이 좋습니다. userInfo 대신 `users` 라는 테이블명이 더 명확해보입니다.
- password 컬럼도 추가해주면 좋을 것 같습니다.
- email 또는 유저의 계정, 핸드폰 번호는 중복 저장이 불가능하게 unique 속성을 추가해주면 좋을 것 같습니다.

다음 예시를 참고하여 수정해주세요~!
```suggestion
CREATE TABLE users (
  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
  name VARCHAR(100) NOT NULL,
  age INT NULL,
  email VARCHAR(100) NOT NULL,
  password VARCHAR(255) NOT NULL,
  created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
  updated_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
  CONSTRAINT users_ukey UNIQUE (email)
);
```